Software Development Questions

Luckyjoseph opened the pull request:
The uploaded file contains all my curated questions on Software Development for the 3MTT program.

Luckyjoseph committed:
My 3MTT Software Development question

The uploaded file contains all my curated questions on Software Development for the 3MTT program.

talktonok reviewed (changes requested):
Questions are not in a tabular form they are not in the right format. please rectify.

Luckyjoseph commented:
Hey there @talktonok , thanks for review.
I am not sure if we are requested to curate our questions in a tabular form for software development.
But i will check it out.

talktonok commented:
> Hey there @talktonok , thanks for review.
> I am not sure if we are requested to curate our questions in a tabular form for software development.
> But i will check it out.
> 

Thank you @Luckyjoseph, please do go true the guidelines once more. For more clarity

Paul-Ben commented:
> Hey there @talktonok , thanks for review. I am not sure if we are requested to curate our questions in a tabular form for software development. But i will check it out.

Hi @Luckyjoseph, good work. but the observation by @talktonok is valid and i think it was stated in the ReadMe document. just go through it again. it may be a lot of work but my brother i encourage you to try. im currently working on mine

udofia2 commented:
This is a great work. I am ready to assist with the refactoring, should my assistant be needed

EmmanuelTheCoder commented:
@Luckyjoseph reach out to @udofia2 if you need help revamping your questions to ensure it conforms to our template. @udofia2  is always willingly to help.

EmmanuelTheCoder commented:
Plus, it will be great if you can add more questions to it.

Luckyjoseph committed:
Updated software development question

I just updated the software development question to follow all the pattern indicated in the guide.

Luckyjoseph commented:
Hey there @udofia2 @EmmanuelTheCoder 
I just added a new commit with the updated questions following all the necessary patterns. Please, help me have a look.

EmmanuelTheCoder commented:
This is great @Luckyjoseph. Thanks.
